### PR TITLE
Add managed by property to resource group

### PR DESCRIPTION
**Pull request by ghost**
We have added a new property named 'ManagedBy' on the resource group. As part of this change, we have updated API version to 2016-09-01.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@laveeshb-MSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (laveeshb). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by TianoMS** [comment]
Looks good to me.

**Comment by vivsriaus** [comment]
@laveeshb-MSFT Thanks for pushing this through to github!

File: arm-resources/resources/2016-09-01/swagger/resources.json [hunk 1]
```diff
@@ -0,0 +1,2509 @@
...
+          "404": {
+            "description": ""
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Deployments"
+        ],
+        "operationId": "Deployments_CreateOrUpdate",
```
Comment (hovsepm) [inline arm-resources/resources/2016-09-01/swagger/resources.json]: @laveeshb-MSFT with this type of modeling SDK users will **have to** provide value for all required fields when they will do Update operation (in your Deployment model Mode is marked as required property)

**Comment by hovsepm** [comment]
Talked offline. My comment will be addressed in a separate PR by @vivsriaus.

**Pull request merged**